classes/Currency.php

```diff
@@ -593,6 +671,30 @@ public static function getIdByIsoCode($isoCode, $idShop = 0)
         return Cache::retrieve($cacheId);
     }
 
+    /**
+     * Get Currency ID by numeric ISO code.
+     *
+     * @param int $numericIsoCode ISO code
+     * @param int $idShop Shop ID
+     *
+     * @return int Currency ID
+     */
+    public static function getIdByNumericIsoCode($numericIsoCode, $idShop = 0)
```

Comment:
```suggestion
    public static function getIdByNumericIsoCode($numericIsoCode, $idShop = null)
```
We don't have a shop id with 0, better to have a null data instead.
lso, we use +, you can type hint parameters :wink: